tests/test_parser.py

```diff
@@ -74,4 +74,3 @@ def test_fizzbuzz():
 if __name__ == "__main__":  # For testing outside of the workflow
```

Comment:
This needs to be removed too btw.

Reply:
yesn't

Reply:
wdym

Reply:
this is a general thing in python programs

Reply:
not with pytest it ain't